cli/command/formatter/stats.go

```diff
@@ -139,7 +141,18 @@ type containerStatsContext struct {
 
 func (c *containerStatsContext) Container() string {
 	c.AddHeader(containerHeader)
-	return c.s.Name
+	return c.s.Container
+}
+
+func (c *containerStatsContext) Names() string {
```

Comment:
@crosbymichael I asked myself the same question :sweat_smile:. I did it to *mimic* the `docker ps --format` which has `Names` too, I'm fine to rename it to `Name` 👼